### ISSUE TITLE
Temporary random bug with text editor

### ISSUE THREAD

**Issue by jluini** [godotengine/godot#38837]
**Godot version:**
v3.2.stable.official

**OS/device including version:**
Windows 7 ultimate (version 6.1.7601 Service Pack 1 compilation 7601)

**Issue description:**
Sometimes when I open a text editor or switch to it I get it in a different "mode". In that "mode" I see less menu entries (only "File" and "Debug" instead of the five usual File/Search/Edit/Go To/Debug) and I'm not able to do some actions (e.g. ctrl-F for find doesn't work, and that's why I realize it's bugged). It apparently works fine apart from that two symptoms.

It's a very small issue because I simply switch to another text editor (or to 2d mode and back to script mode) and the problem is gone.

I'm not able to reproduce it, looks like random to me; it's more frequent when I just opened the Godot editor, though.

**Steps to reproduce:**
Can't reproduce it.

**Comment by Calinou** [comment]
Can you take a screenshot and upload it next time you manage to reproduce the bug?

**Comment by jluini** [comment]
Yes, I will. Sorry for opening the issue without the screenshot (I had taken it and accidentally lost it, then decided to open the issue nevertheless).

I'm pretty sure it will be a normal screen with only File and Debug menues though.

**Comment by jluini** [comment]
I got the bug, this time after doing _Close Other Tabs_ within the text editor (it probably doesn't have nothing to do with the issue though, because I got the bug in another circumstances too).

Here goes the screenshot. I recorded a video too, it's [here](https://drive.google.com/open?id=1wfsYsfd_MabDPcK2M3ccKETt-APwGCJK) in **mkv** format.

![godot_bug](https://user-images.githubusercontent.com/7409047/82496449-20559000-9ac3-11ea-8168-0c9a4f9ad0fe.jpg)

**Comment by jluini** [comment]
Got it two more times, always after _Close Other Tabs_.

I still can't reproduce it though.

**Comment by jluini** [comment]
Lately I'm getting it only after _Close Other Tabs_ and sometimes with very high frequency (looking almost reproducible) so I changed the issue title. You can see in [this video](https://drive.google.com/open?id=1IDs4zmvRY2s1qY3rmNhmfD5jh0bKIXwG) how I got it three times in a row.

But it doesn't happen always, and I'm pretty sure I got it in other circumstances too.

**Comment by jluini** [comment]
I changed the issue description as I'm getting the same bug in Linux (Ubuntu 20.04, 3.2.1.stable.official).
